tests/integration/_cases/logs/logs-list-with-custom-max-rows.trycmd

```diff
@@ -0,0 +1,5 @@
+$ sentry-cli logs list --org wat-org --project 12345 --max-rows 50
+? success
++------------------+---------------------+----------+----------------------+---------------------+
+| Item ID          | Timestamp           | Severity | Message              | Trace               |
++------------------+---------------------+----------+----------------------+---------------------+ 
```

Comment:
This output looks wrong. I would expect it to be the same as `logs-list-with-data.trycmd`. Why is it different?